packages/react/src/List/List.module.css

```diff
@@ -0,0 +1,35 @@
+.List {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  list-style: none;
+  gap: var(--base-size-8);
+  padding-inline-start: 0px;
+  font-family: var(--brand-body-fontFamily);
+  font-weight: var(--brand-text-weight-400);
+  font-size: var(--brand-text-size-200);
+  line-height: var(--brand-text-lineHeight-400);
+  color: var(--brand-color-text-default);
+}
+
+ol.List {
+  counter-reset: li;
+}
+
+.ListItem {
```

Comment:
If ListItem is a separate component, this should be in a dedicated stylesheet for it.

Reply:
How about the issue that they are linked for the OL and would need access to both the List and ListItem class in one file. Would you put that declaration in the ListItem file?

Reply:
> How about the issue that they are linked for the OL and would need access to both the List and ListItem class in one file. Would you put that declaration in the ListItem file?

What did you mean by linked? I see ListItem declared a separate component, and there doesn't seem to be a reference to `.ListItem` in the `UnorderedList` component. We typically create stylesheets per components or pattern, and given `UnorderedList `, `OrderedList` and `ListItem` are all separate, I would create individual stylesheets for each of them. Related comment about locally scoping `ListItem` so maybe you only need two stylesheets instead.

Reply:
The CSS needs access to the List class as well as the ListItem class in the same declaration to elegantly create the numbers in the ordered list. If you split them the CSS breaks. How would you approach this kind of CSS?

Reply:
Also wouldn't repeating CSS declarations bloat the CSS bundles when both are in use? Or does it remove duplicates?

Reply:
They should get deduped now but I haven't tested it recently. Fairly easy to fix using postcss if it doesn't. Either way, we should leave that to Webpack to figure out through the dependency tree, as it will become important to reduce side-effects when we enable code splitting.

Reply:
> The CSS needs access to the List class as well as the ListItem class in the same declaration to elegantly create the numbers in the ordered list. If you split them the CSS breaks. How would you approach this kind of CSS?

That's a good question 🤔 

So first, I don't think this stylesheet should be called `List`, as there's no `List` component anymore. Perhaps `list-shared.module.css.` or similar, and should live in the root `/src/list/list-shared.css`

Then you have two more stylesheets for `UnorderedList.module.scss` and `OrderedList.module.css`, and they only have styles pertaining to their respective component files.

As part of this change, I think the folder structure should generally change to be more like `src/forms`. 

```ascii
.
├── src
├── ...
└── lists/
    ├── index.ts
    ├── list.shared.module.css
    ├── UnorderedList/
    │   ├── UnorderedList.module.css
    │   ├── UnorderedList.tsx
    │   └── index.ts
    └── OrderedList/
        ├── OrderedList.module.css
        ├── OrderedList.tsx
        └── index.ts
```

Reply:
Updated in next push